packages/admin/cms-admin/src/dam/blocks/DamFileDownloadLinkBlock.tsx

```diff
@@ -0,0 +1,144 @@
+        const ret: DamFileDownloadLinkBlockData = {
+            openFileType: output.openFileType,
+        };
+
+        if (output.fileId === undefined) {
+            return ret;
+        }
+
+        const { data } = await apolloClient.query<GQLDamFileDownloadLinkFileQuery, GQLDamFileDownloadLinkFileQueryVariables>({
+            query: gql`
+                query DamFileDownloadLinkFile($id: ID!) {
+                    damFile(id: $id) {
+                        id
+                        name
+                        fileUrl
+                    }
+                }
+            `,
+            variables: { id: output.fileId },
+        });
+
+        const { damFile } = data;
+
+        ret.file = {
+            id: damFile.id,
+            name: damFile.name,
+            fileUrl: damFile.fileUrl,
+        };
+
+        return ret;
+    },
+
+    definesOwnPadding: true,
+
+    AdminComponent: ({ state, updateState }) => {
+        return (
+            <>
+                <BlocksFinalForm<{
+                    openFileType: DamFileDownloadLinkBlockData["openFileType"];
+                    file?: DamFileDownloadLinkBlockData["file"];
+                }>
+                    onSubmit={(newValues) => {
+                        updateState({
+                            file: newValues.file ?? undefined,
+                            openFileType: newValues.openFileType,
+                        });
+                    }}
+                    initialValues={{
+                        file: state.file,
+                        openFileType: state.openFileType ?? "DOWNLOAD",
+                    }}
+                >
+                    {state.file !== undefined ? (
+                        <AdminComponentPaper disablePadding>
+                            <Box padding={3}>
+                                <Typography variant="subtitle1">{state.file.name}</Typography>
+                                <Typography variant="body1" color="textSecondary">
+                                    <DamPathLazy fileId={state.file.id} />
+                                </Typography>
+                            </Box>
+                            <Divider />
+                            <AdminComponentButton startIcon={<Delete />} onClick={() => updateState({ file: undefined, openFileType: "NEW_TAB" })}>
+                                <FormattedMessage {...messages.empty} />
+                            </AdminComponentButton>
+                        </AdminComponentPaper>
+                    ) : (
+                        <Field name="file" component={FileField} fullWidth />
+                    )}
+
+                    <Divider />
+                    <AdminComponentPaper>
+                        <Field
+                            name="openFileType"
+                            fullWidth
+                            label={<FormattedMessage id="blocks.damFileDownloadLink.openFileType" defaultMessage="Open File Type" />}
```

Comment:
I don't really like this label. I consulted with UX and we agreed on this version:

<img width="379" alt="Bildschirmfoto 2024-04-11 um 14 15 10" src="https://github.com/vivid-planet/comet/assets/13380047/3f347f1b-7afb-429c-a29c-3b2c1e0248cf">